src/components/Cell/Cell.scss

```diff
@@ -1,22 +1,27 @@
 .cell {
     background-color: white;
     border-radius: 5px;
-    border: solid white 3px;
     margin: 10px;
     height: 2.5em;
     width: 2.5em;
-    &:not(.disabled):hover {
+    &:not(.disabled):not(.disable-scale):hover {
         transform: scale(1.3);
     }
+    position: relative;
+    backface-visibility: hidden;
 }
 
 .disabled {
     background-color: #786961;
     transform: scale(.75);
-    border: none;
+    margin: 12px;
```

Comment:
nicely done re-centering the disabled cells. not noticeable if you aren't looking for it!